tools/kochdocs.nim

```diff
@@ -14,7 +14,10 @@ const
   webUploadOutput = "web/upload"
 
 var nimExe*: string
-const allowList = ["jsbigints.nim", "jsheaders.nim", "jsformdata.nim", "jsfetch.nim", "jsutils.nim"]
+const allowList = [
```

Comment:
how about doing the reverse, use a `nonJsList` (possibly empty currently) and otherwise assume that a module that starts with js is js-specific

Reply:
On follow up PR...